rust/datafusion/src/logical_plan/expr.rs

```diff
@@ -265,6 +277,7 @@ impl Expr {
                 ..
             } => Ok(left.nullable(input_schema)? || right.nullable(input_schema)?),
             Expr::Sort { ref expr, .. } => expr.nullable(input_schema),
+            Expr::Between { .. } => Ok(true),
```

Comment:
Isn't BETWEEN nullable if the underlying expression is nullable?

Reply:
@andygrove good catch. i have updated the PR.